orchestration/dagster_orchestration/hca_orchestration/solids/load_hca/stage_data.py

```diff
@@ -5,6 +5,10 @@
 from google.cloud.bigquery import Dataset
 
 
+class HcaStagingDatasetName(str):
```

Comment:
The load data files pipeline will be passing many `str` parameters around, I am using this to bring some semblance of type safety to those parameters. Not 100% sure on the placement in this file, if folks have opinions.

Reply:
i think custom types fit best in `hca_orchestration/support/typing.py`, with the potential to split that into subfiles in the `typing` directory instead if that file becomes too large or too disparate in function